UPGRADING

```diff
@@ -145,6 +145,11 @@ PHP 8.0 UPGRADE NOTES
     as a string instead of an ASCII codepoint. The previous behavior may be
     restored with an explicit call to chr().
 
+- PCRE:
+  . When passing invalid escape sequences they are no longer intepreted as
+    literals. This behaviour previously required the X modifier - which has
+    been removed as well.
```

Comment:
which is now ignored

Reply:
agreed - I've amended this to the second commit